content/pages/volunteers.md

```diff
@@ -0,0 +1,96 @@
+Title: Volunteer Opportunities
+Category: pages
+Summary: Volunteer opportunities available with WiCS undergraduate.
+
```

Comment:
You will need to add a `status: hidden` metadata line here, otherwise we break the navbar which is a continuing issue we should probably fix